### DISCUSSION TITLE
Rückgängig gemachte Aufgabe sofort wieder zur Lösung anbieten

### DISCUSSION THREAD

**Discussion by OpenGreenStreet** [streetcomplete/StreetComplete#2485] [Ideas]
Ich würde mir wünschen, dass rückgängig gemachte Aufgabe sofort wieder zur Lösung angeboten werden. ("walk around" ist meiner Meinung nach das Löschen des Caches, aber das dauert mir zu lange . . . :-)

**Comment by riQQ** [comment]
Siehe https://github.com/streetcomplete/StreetComplete/issues/2418

**Reply by OpenGreenStreet** [reply]
prima, Danke für den Hinweis
